Integrations/GoogleKeyManagementService/GoogleKeyManagementService.py

```diff
@@ -81,28 +82,80 @@ def arg_dict_creator(string: str):
     split_string = string.split(',')
     arg_dict = {}
     for section in split_string:
-        key, value = section.split(':', 1)
-        arg_dict[key] = value
+        section_key, section_value = section.split(':', 1)
+        arg_dict[section_key] = section_value
 
     return arg_dict
 
 
-def key_context_creation(res: Any) -> Dict:
+def clear_label_commas(labels: Any):
+    """When extracted from the response - the labels return with an added commas and spaces
+    This function removes these commas.
+
+    Args:
+        labels(Dict): a dictionary of labels as returned from the response.
+
+    Returns:
+        the label dictionary without the commas and spaces.
+    """
+    if not labels:
+        return None
+
+    cleared_labels = {}  # type:Dict
+    for label in labels.keys():
```

Comment:
`for label in labels:`

can you give an example of the original labels value?
why the magic numbers? 1, 2, -1

Reply:
will address that